sbt-gcs-plugin/src/main/scala/org/latestbit/sbt/gcs/artifactregistry/GcsArtifactRegistryUrlConnection.scala

```diff
@@ -25,7 +25,8 @@ import scala.util.Try
 class GcsArtifactRegistryUrlConnection( googleHttpRequestFactory: HttpRequestFactory, url: URL )( implicit
     logger: Logger
 ) extends HttpURLConnection( url ) {
-  private final val genericUrl = GcsArtifactRegistryGenericUrlFactory.createFromUrl( url )
+  private final val genericUrl           = GcsArtifactRegistryGenericUrlFactory.createFromUrl( url )
+  private var props: Map[String, String] = Map.empty
```

Comment:
It seems HttpURLConnection/URLConnection from JVM already has the map, so there is no need to repeat that here I think?